src/app/dashboard-widgets/applications-widget/applications-list-item-details/applications-list-item-details.component.less

```diff
@@ -0,0 +1,104 @@
+@import (reference) '../../../../assets/stylesheets/shared/osio.less';
+
+.pipeline {
+  border-top: 1px solid @color-pf-black-200;
+  display: flex;
+  flex-wrap: nowrap;
+  height: 100%;
+  a {
+    &.stack-reports-btn {
+      display: none;
+    }
+  }
+}
+
+.pipeline .pipeline-stage {
+  flex: 0 0 auto;
+  min-height: 96px;
+  padding: 15px 10px 30px;
+  position: relative;
+  width: 100%;
+  &.pipeline-stage-min {
+    padding: 51px 10px 30px;
+  }
+}
+
+.pipeline .pipeline-stage:before {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  line-height: 1;
+  font-size: 22px;
+  text-align: center;
+  color: #c4c4c4;
```

Comment:
change to a PatternFly color variable

Reply:
This color comes from the pipelines page. The @color-pf-black-400 seems to match.